set.mm

```diff
@@ -50034,14 +50033,22 @@ we define it to be empty in this case (see ~ opprc1 , ~ opprc2 , and
        Axiom of Regularity for its justification and is not commonly used.  A
        definition that also works for proper classes is ` <. A , B >. ` _4
        ` = ( ( A X. { (/) } ) u. ( B X. { { (/) } } ) ) ` , justified by
-       ~ opthprc .  If we restrict our sets to nonnegative integers, an ordered
-       pair definition that involves only elementary arithmetic is provided by
-       ~ nn0opthi .  An ordered pair of real numbers can also be represented by
-       a complex number as shown by ~ cru .  Kuratowski's ordered pair
-       definition is standard for ZFC set theory, but it is very inconvenient
-       to use in New Foundations theory because it is not type-level; a common
-       alternate definition in New Foundations is the definition from [Rosser]
-       p. 281.
+       ~ opthprc .  Nearly at the same time as Norbert Wiener, Felix Hausdorff
+       proposed the following definition in "Grundz&uuml;ge der Mengenlehre"
+       ("Basics of Set Theory"), p. 32, in 1914: ` <. A , B >. ` _5
+       ` = { { A , O } , { B , T } } ` .  Hausdorff used ` 1 ` and ` 2 `
+       instead of ` O ` and ` T ` , but actually any two different fixed sets
+       will do (e.g., ` O = (/) ` and ` T = { (/) } ` , see ~ 0nep0 ).
+       Furthermore, Hausdorff demanded that ` O ` and ` T ` are both different
+       from ` A ` as well as ` B ` , which is actually not necessary (at least
+       not in full extend), see ~ opthhausdorff0 and ~ opthhausdorff .  If we
```

Comment:
extend --> extent
(but maybe better: "at least not for most applications")

OK. I haven't checked the original sources, but it sounds indeed like one of the historically proposed alternatives for the ordered pair definition (among yet others). On the other hand, set.mm is not meant as a history reference, so this one is fine, but no need to put too many of these in my opinion.

Reply:
OK, typo will be corrected.

Of course not *all* historic variants should be discussed (and proven) in set.mm, but the important or interesing ones should be.

Reply:
An important variant, which also works for proper classes, is defined in my mathbox (section "Tuples of classes").

Reply:
It's a shame that I do not know all the treasures which are buried in the mathboxes... Yes, I agree that this a more important variant (especially the variant ~bj-2uplth of ~opth is very interesting), and I would have proposed to move it to main, too, unless there were a lot of new definitions involved (for my Hausdorff variant, no new definition is needed - its just a justification).

Reply:
Typo fixed.